rbac/rbac/settings.py

```diff
@@ -348,6 +348,7 @@
 # Seeding Setup
 ROLE_SEEDING_ENABLED = ENVIRONMENT.bool('ROLE_SEEDING_ENABLED', default=True)
 GROUP_SEEDING_ENABLED = ENVIRONMENT.bool('GROUP_SEEDING_ENABLED', default=True)
+MAX_SEED_THREADS = ENVIRONMENT.int('MAX_SEED_THREADS', default=None)
```

Comment:
Does it make sense to give a definite default, versus leaving it open-ended?

Reply:
@j00bar - Maybe? I went back and forth on this yesterday. When it's `None`, it will use the default value, which is the current behavior. For now, I figured we'd just leave the behavior as is, while closing the connections explicitly. So the `max_workers` option would just be a no-op currently. 

Then if we want/need to limit the concurrency further in the future we could set `MAX_SEED_THREADS`.

I'm open to suggestions though, if we feel like we should limit it out of the gate! Initially that was the plan, since that was one way to prevent connections from being exhausted, but with closing the connections it removes that necessity. I ended up leaving it in more for future flexibility.

Reply:
Ah, okay - cool. I think that sounds fine - I just wouldn't want the default to exhaust the connections and it sounds like it won't.